.github/workflows/pr-build.yaml

```diff
@@ -46,7 +46,17 @@ jobs:
           -Dorg.gradle.jvmargs=-Xmx5120M \
           -PsourceMaven=https://development.galasa.dev/gh/maven-repo/maven/ \
           -PcentralMaven=https://repo.maven.apache.org/maven2/ \
-          -PtargetMaven=${{ github.workspace }}/repo 
+          -PtargetMaven=${{ github.workspace }}/repo \
+          --info

```

Comment:
Sorry nitpicky but we already have the `--info` flag in this command not sure if this will cause issues